talisker/__init__.py

```diff
@@ -48,11 +48,18 @@ def initialise(env=os.environ):
     return config
 
 
+ACTIVE = set(['true', '1', 'yes'])
+
+
 def get_config(env=os.environ):
     """Load talisker config from environment"""
+    devel = env.get('DEVEL', '').lower() in ACTIVE
     return {
+        'devel': devel,
         'debuglog': env.get('DEBUGLOG'),
-        'devel': sys.stderr.isatty() or 'DEVEL' in env,
+        'color': (
+            devel and sys.stderr.isatty() and 'TALISKER_NO_COLOR' not in env
```

Comment:
Would it be better to have a `TALISKER_COLOR` env var that can be either "yes" or "no" and, if present, use that in preference to any tty detection? That would mean you could also override a false -ve from `isatty()`.